samples/bluetooth/mesh/light/boards/thingy53_nrf5340_cpuapp.conf

```diff
@@ -6,33 +6,8 @@
 ################################################################################
 # Application overlay - Thingy:53 DFU
 
-CONFIG_MCUMGR=y
-CONFIG_MCUMGR_SMP_BT=y
-CONFIG_MCUMGR_SMP_BT_AUTHEN=n
-CONFIG_MCUMGR_SMP_BT_CONN_PARAM_CONTROL=y
-
-CONFIG_MCUMGR_CMD_IMG_MGMT=y
-CONFIG_MCUMGR_CMD_OS_MGMT=y
-
-# Enable MCUmgr Packet Reassembly feature over Bluetooth and its
-# configuration dependencies. MCUmgr buffer size is optimized to fit one SMP
-# packet divided into five Bluetooth Write Commands, transmitted with the
-# maximum possible MTU value: 498 bytes.
-CONFIG_MCUMGR_SMP_REASSEMBLY_BT=y
-CONFIG_MCUMGR_BUF_SIZE=2475
-CONFIG_OS_MGMT_MCUMGR_PARAMS=y
-
-# Enable custom SMP request to erase settings partition.
-CONFIG_MCUMGR_GRP_ZEPHYR_BASIC=y
-CONFIG_MCUMGR_GRP_BASIC_CMD_STORAGE_ERASE=y
-
-# Increase system workqueue stack size to prevent stack overflows while handling
-# SMP requests.
-CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
```

Comment:
I am not sure if switching to using default system workqueue stack size is safe now (keep in mind that workqueue could be also used for other purposes). Maybe we could leave it as it is for the release?

Reply:
Yes, it is safer to leave it as it is. However, I will remove the comment as it is no longer necessary to have it enabled for MCUmgr module